istio/istio.spec

```diff
@@ -377,8 +377,7 @@ tar zxf %{SOURCE0} -C ISTIO/src/istio.io/istio --strip=1
 
 cp %{SOURCE1} ISTIO/src/istio.io/istio/.istiorc.mk
 
-sed -i "s|istio.io/pkg/version\.buildVersion=.*|istio.io/pkg/version.buildVersion="%{version}"|" %{SOURCE2}
-cp %{SOURCE2} ISTIO/src/istio.io/istio/buildinfo
+sed "s|istio.io/pkg/version\.buildVersion=.*|istio.io/pkg/version.buildVersion=Maistra_%{version}|" %{SOURCE2} > ISTIO/src/istio.io/istio/buildinfo
```

Comment:
From my understanding, this doesn't fix the requested problem. It only means that instead of pulling 1.1.0 out, now we'll have Maistra_1.1.0, giving Istio Maistra_1.1.0 for a version.

Reply:
Per conversation on IRC, product/Maistra will list versions as: 

Istio Maistra_1.1.0 and Istio OSSM_1.1.0

Reply:
Just a note for readers: This `Istio` string above is only shown in Kiali's about dialog.

When running `<binary> version` you'll get something like 
```
$ ./pilot-discovery version
version.BuildInfo{Version:"Maistra_1.1.0", GitRevision:...
```